src/core/abr/representation_chooser.ts

```diff
@@ -280,6 +296,31 @@ export default class RepresentationChooser {
   private estimator : BandwidthEstimator;
   private _initialBitrate : number;
 
+  /**
+   * Information about frames (total decoded and dropped)
+   * Emitted at each clock event.
+   */
+  private lastDroppedFrames: number = 0;
+  private lastTotalFrames: number = 0;
```

Comment:
The previous twos could be better if already provided by the clock.

We could even make use of rxjs' scan operator or something like that to calculate this value without managing a state.